src/barnetilsyn/BarnetilsynInformasjon.tsx

```diff
@@ -34,20 +35,25 @@ export const BarnetilsynInformasjon: React.FC<InformasjonProps> = ({
   const [locale] = useSpråkContext();
   const intl = useLokalIntlContext();
 
-  const erForrigeSøknadBesvartPåSammeSpråkSomErValgt = (
+  const finnesForrigeSøknadOgErBesvartPåSammeSpråkSomErValgt = (
     forrigeSøknad: ForrigeSøknad
   ) => {
-    return (
-      locale === 'nb' &&
-      forrigeSøknad.sivilstatus?.årsakEnslig?.label ===
-        hentTekst('sivilstatus.spm.begrunnelse', intl)
-    );
+    if (
+      stringHarVerdiOgErIkkeTom(forrigeSøknad.sivilstatus?.årsakEnslig?.label)
```

Comment:
To ting;
Burde ikke `forrigeSøknad` være nullable i input her?

Og kan vi bare sjekke `if(forrigeSøknad)` istedenfor?

Reply:
![Screenshot 2024-02-13 at 15 08 40](https://github.com/navikt/familie-ef-soknad/assets/402915/06df3e90-af91-43d2-8c68-158faa9f88b0)